contrib/babelfishpg_tsql/src/procedures.c

```diff
@@ -2427,7 +2427,7 @@ gen_sp_droprolemember_subcmds(const char *user, const char *member)
 }
 
 static void 
-update_bbf_server_options(char *servername, char *optname, char *optvalue, bool isInsert)
+update_bbf_server_options(char *servername, char **optname, char **optvalue, bool isInsert)
```

Comment:
The hardcoded arrays can be defined globally (since these won't change) and we can use those every time an insert happens. We can ignore the optname and optvalue arguments and just do a catalog insert using the default values.